ui/src/views/Rules/ListRules/index.jsx

```diff
@@ -1285,12 +1370,33 @@ function ListRules(props) {
   }, [hashQuery, filteredRulesCount]);
 
   return (
-    <Dashboard title="Rules">
+    <Dashboard
+      title={
+        rewindDate ? `Rules @ ${rewindDate.toString().split('(')[0]}` : 'Rules'
+      }>
       {isLoading && <Spinner loading />}
       {error && <ErrorPanel fixed error={error} />}
       {!isLoading && productChannelOptions && (
         <Fragment>
           <div ref={searchFieldRef} className={classes.options}>
+            <DateTimePicker
+              disableFuture
+              clearable
+              inputVariant="outlined"
+              label="Rewind to..."
```

Comment:
todo: please align this control and label with the others on the page. It currently sits slightly below the others:
![image](https://github.com/mozilla-releng/balrog/assets/49649/9e5b687a-9277-48b6-8a19-847608db9663)